pylint/test/unittest_checker_python3.py

```diff
@@ -203,6 +203,8 @@ def test_dict_methods_in_iterating_context(self):
             '[x for x in {}]',
             'func({})',
             'a, b = {}',
```

Comment:
these tests are currently somewhat nonsensical.  This code expands to `a, b = {}.keys` when it really should be expanding to `a, b = {}.keys()` -- though fixing this causes the test to fail so I suspect something worse is going on here that I don't quite understand?

Reply:
Thanks for finding this bug! I just pushed a fix, it was failing because when we were passing the `keys()` into a function call, but only some functions are exempt from this check, while the one from the test wasn't.

Reply:
makes sense -- I was in a hurry to push this branch and didn't look into it too closely 😆